### ISSUE TITLE
Unneeded code?

### ISSUE THREAD

**Issue by jernst** [hoopsurfer/pidesktop#19]
`pd-reboot.py` seems, at some point, have been doing this:
```
GPIO.output(31,GPIO.HIGH)
```
but that's now commented out and all the rest is just print statements. Which means the entire `pidesktop-reboot.service` may not be needed any more?

**Comment by hoopsurfer** [comment]
true, actually that moved to the gpio command added to /boot/config.txt which works so much better in my testing.

I left the reboot service because it provides a useful hook, some messages in the log and hoping I could get at the firmware soon and make it more useful, so far that hasn't happened.
